mkdocs.yml

```diff
@@ -43,7 +43,7 @@ nav:
     - 'Step 4: Configurations': 'operation/loop-settings/configurations.md'
     - 'Step 5: Loop Services': 'operation/loop-settings/services.md'
     - 'Loop Displays': 'operation/loop-settings/displays.md'
-    - 'RileyLink menu': 'operation/loop-settings/rileylink.md'
+    - 'Device Menu': 'operation/loop-settings/rileylink.md'
```

Comment:
I know that this isn't specific to the original RileyLink, but I think that I'd stick with using "RileyLink Menu" for the pulldown menus labels for consistency with "Step 5 Order a RileyLink" and "RileyLink FAQs".